app/src/test/contrib/components/Responsibilities/Uploads/UploadFormTests.tsx

```diff
@@ -1,73 +1,68 @@
 import * as React from "react";
-import { expect } from "chai";
-import { UploadForm } from "../../../../../main/contrib/components/Responsibilities/BurdenEstimates/UploadForm";
-import { shallow, ShallowWrapper } from "enzyme";
-import {mockBurdenEstimateSet, mockResponsibility, mockScenario} from "../../../../mocks/mockModels";
-import { setupMainStore } from "../../../../mocks/mocks";
-import { BurdenEstimateSet } from "../../../../../main/shared/models/Generated";
-import { Sandbox } from "../../../../Sandbox";
+import {expect} from "chai";
+import {UploadForm} from "../../../../../main/contrib/components/Responsibilities/BurdenEstimates/UploadForm";
+import {shallow, ShallowWrapper} from "enzyme";
+import {mockBurdenEstimateSet, mockResponsibility} from "../../../../mocks/mockModels";
+import {Sandbox} from "../../../../Sandbox";
 import {mockFetcher} from "../../../../mocks/mockRemote";
 import {CurrentEstimateSetSummary} from "../../../../../main/contrib/components/Responsibilities/Overview/List/CurrentEstimateSetSummary";
 
 const buttonStyles = require("../../../../../main/shared/styles/buttons.css");
-const messageStyles = require("../../../../../main/shared/styles/messages.css");
 
 describe('UploadForm', () => {
     let rendered: ShallowWrapper<any, any>;
     const sandbox = new Sandbox();
     before(() => mockFetcher(Promise.resolve(null)));
 
-    function setUpComponent(canUpload: boolean,
```

Comment:
We don't actually need to set up the store for this component so this function was just obscuring the test logic.